### PR TITLE
restructering docu to prepare for more on homological algebra

### PR DESCRIPTION
**Pull request by wdecker**

### CODE REVIEW

File: docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/homological_algebra.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+```
+
+## Hom and Ext
+
+```@docs
+hom(M::ModuleFP, N::ModuleFP, alg::Symbol=:maps)
+```
+
+```@docs
+element_to_homomorphism(f::ModuleFPElem)
```
Comment (afkafkafk13) [inline docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/homological_algebra.md:53]: Please discuss the name of the function with @fieker before building further functionality on top of it. He has an opinion on function names A_to_B

File: docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/homological_algebra.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+```
+
+```@docs
+homomorphism_to_element(H::ModuleFP, phi::ModuleFPHom)
```
Comment (afkafkafk13) [inline docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/homological_algebra.md:57]: same comment as in line 53

**Review by afkafkafk13 — COMMENTED** [review]
see comments on function names

File: docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/homological_algebra.md (hunk 2 above)
Comment (wdecker) [reply docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/homological_algebra.md:57]: These names and their use are old and not part of this PR. The relevant functions have only be moved from one file to another one.

**Review by afkafkafk13 — APPROVED** [review]

**Pull request merged**